### PR TITLE
deal (badly) with non-trivial Schur index

### PR DESCRIPTION
**Pull request by fieker**
needs Thomas PR #1434  for the char table

### CODE REVIEW

File: experimental/GModule/GModule.jl [hunk 1]
```diff
@@ -207,15 +207,37 @@ function irreducible_modules(::Type{AnticNumberField}, G::Oscar.GAPGroup; minima
...
+      end
+      if d !== nothing && d*degree(k) == degree(base_ring(V))
         push!(res, V)
-      else
+      elseif d == 1
         @vprint :MinField 1 "Going from $(degree(base_ring(V))) to $(degree(k))"
         Vmin = gmodule_over(m, V)
         push!(res, Vmin)
+      else
+        if d === nothing
```
Comment (fingolfin) [inline experimental/GModule/GModule.jl:221]: So when is `nothing` returned and why?

**Comment by fieker** [comment]
On Fri, Jul 01, 2022 at 04:54:12AM -0700, Max Horn wrote:
> @fingolfin commented on this pull request.
> 
> 
> 
> >          @vprint :MinField 1 "Going from $(degree(base_ring(V))) to $(degree(k))"
>          Vmin = gmodule_over(m, V)
>          push!(res, Vmin)
> +      else
> +        if d === nothing
> 
> So when is `nothing` returned and why?
> 
schur_index can return nothing - should rather be (my opinion)
error("not implemented yet")

**Comment by fingolfin** [comment]
```
Experimental.gmodule: Error During Test at /home/runner/work/Oscar.jl/Oscar.jl/test/Experimental/gmodule-test.jl:1
  Got exception outside of a @test
  MethodError: no method matching known_class_fusion(::Nothing, ::Oscar.GAPGroupCharacterTable)
Closest candidates are:
  known_class_fusion(::Oscar.GAPGroupCharacterTable, ::Oscar.GAPGroupCharacterTable) at ~/work/Oscar.jl/Oscar.jl/src/Groups/group_characters.jl:1048
```

**Comment by thofma** [comment]
Now we have
```
Experimental.gmodule: Error During Test at /home/runner/work/Oscar.jl/Oscar.jl/test/Experimental/gmodule-test.jl:1
  Got exception outside of a @test
  cannot determine the Schur index with the currently used criteria
```
which is good? Or bad?

**Comment by fieker** [comment]
On Wed, Jul 06, 2022 at 07:15:37AM -0700, Tommy Hofmann wrote:
> Now we have
> ```
> Experimental.gmodule: Error During Test at /home/runner/work/Oscar.jl/Oscar.jl/test/Experimental/gmodule-test.jl:1
>   Got exception outside of a @test
>   cannot determine the Schur index with the currently used criteria
> ```
> which is good? Or bad?
> 
It means that instead of an "nothing" there is an error. I'll change the
code to try-catch and Thomas is actually on solving this problem
completely.

**Comment by fingolfin** [comment]
I talked to @ThomasBreuer yesterday and he was still looking into the Schur index algorithms; but there may also be some misunderstanding going on, if I recall correctly (my apologies if I don't) then @ThomasBreuer said he was waiting for something @fieker wanted to do first?!

**Comment by ThomasBreuer** [comment]
@fingolfin No, this was a misunderstanding, I am not waiting for more input by Claus.

**Comment by fingolfin** [comment]
@ThomasBreuer what's the status of this?

**Comment by fingolfin** [comment]
@ThomasBreuer ping?

**Comment by thofma** [comment]
I (again) had to use the computer algebra system that shall not be named to compute a Schur index.

**Comment by ThomasBreuer** [comment]
See pull request #1603 for a workaround.

**Pull request merged**